mac/Keyman4MacIM/Keyman4MacIM/KMAboutWindow/KMAboutWindowController.m

```diff
@@ -49,9 +50,18 @@ - (void)windowDidLoad {
 - (IBAction)configAction:(id)sender {
     // Using `showConfigurationWindow` instead of `showPreferences:` because `showPreferences:` is missing in
     // High Sierra (10.13.1 - 10.13.3). See: https://bugreport.apple.com/web/?problemID=35422518
-    // TODO: This bug should be fixed in 10.13.4, so after that has been available for a few weeks, we
-    // can create a PR for branch mac-revert-high-sierra-showPreferences-workaround and merge it.
-    [self.AppDelegate showConfigurationWindow];
+    // rrb: where available (everywhere but 10.13.1-10.13.3) call showPreferences, otherwise, call our workaround
+    u_int16_t systemVersion = [KMOSVersion SystemVersion];
+    if (0x0AD1 <= systemVersion && systemVersion <= 0x0AD3) // between 10.13.1 and 10.13.3 inclusive
```

Comment:
Can we use defined constants instead of magics please? (Also, what happens when we get to 10.16.0? 😉)